extensions/resteasy-reactive/rest-client-reactive/deployment/src/main/java/io/quarkus/rest/client/reactive/deployment/ClientExceptionMapperHandler.java

```diff
@@ -0,0 +1,105 @@
+package io.quarkus.rest.client.reactive.deployment;
+
+import java.lang.reflect.Modifier;
+
+import javax.ws.rs.Priorities;
+import javax.ws.rs.core.Response;
+
+import org.eclipse.microprofile.rest.client.ext.ResponseExceptionMapper;
+import org.jboss.jandex.AnnotationInstance;
+import org.jboss.jandex.AnnotationTarget;
+import org.jboss.jandex.ClassInfo;
+import org.jboss.jandex.MethodInfo;
+import org.jboss.jandex.Type;
+
+import io.quarkus.gizmo.ClassCreator;
+import io.quarkus.gizmo.ClassOutput;
+import io.quarkus.gizmo.MethodCreator;
+import io.quarkus.gizmo.MethodDescriptor;
+import io.quarkus.gizmo.ResultHandle;
+import io.quarkus.runtime.util.HashUtil;
+
+/**
+ * Generates an implementation of {@link org.eclipse.microprofile.rest.client.ext.ResponseExceptionMapper}
+ * from an instance of {@link io.quarkus.rest.client.reactive.ClientExceptionMapper}
+ */
+class ClientExceptionMapperHandler {
+
+    private final ClassOutput classOutput;
+
+    ClientExceptionMapperHandler(ClassOutput classOutput) {
+        this.classOutput = classOutput;
+    }
+
+    Result generateResponseExceptionMapper(AnnotationInstance instance) {
```

Comment:
it would be nice to have a comment with example generated code

Reply:
Good idea. I'll add one when in the follow up I plan to open with the documentation